extensions/datastores/hbase/src/main/java/mil/nga/giat/geowave/datastore/hbase/operations/BasicHBaseOperations.java

```diff
@@ -38,6 +45,7 @@
 	private final Connection conn;
 	private final String tableNamespace;
 	private final boolean schemaUpdateEnabled;
+	private final HashMap<String, List<String>> coprocessorCache = new HashMap<String, List<String>>();
```

Comment:
perhaps outside of the scope of this issue, but food for thought it may be nice to have something like the Cassandra implementation of KeyspaceStatePool that basically caches stateful data so that it can be used across instances of the Operations.  In other words, if the lifecycle of these operations objects is short-lived the cache could become less useful, but definitely not something that needs immediate attention.